### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.landscape.text.only)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2290]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/#editor)
#### Element path
.tutorialname > .landscape.text.only
#### Snippet
`<span class="ui text landscape only">Flashing Heart</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.15 (foreground color: #fff3e9, background color: #ff8b27, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

tutorial title header
![Screen Shot 2019-07-18 at 2 35 18 PM](https://user-images.githubusercontent.com/5615930/61493789-4e30ec00-a969-11e9-920a-8e7e7f91c309.png)
